components/automate-ui/src/app/modules/infra-proxy/environment-details/environment-details.component.html

```diff
@@ -37,12 +37,16 @@
             <chef-table>
               <chef-thead>
                 <chef-tr>
+                  <chef-th>Cookbook</chef-th>
+                  <chef-th>Operator</chef-th>
                   <chef-th>Version</chef-th>
                 </chef-tr>
               </chef-thead>
               <chef-tbody>
-                <chef-tr *ngFor= "let version of cookbookVersions">
-                  <chef-td  >{{ version }}</chef-td>
+                <chef-tr *ngFor= "let version of cookbookVersions | keyvalue">
+                  <chef-td  >{{ version.key }}</chef-td>
+                  <chef-td  >{{ version.value.split(" ")[0] }}</chef-td>
+                  <chef-td  >{{ version.value.split(" ")[1] }}</chef-td>
```

Comment:
Please refactor the backing code so we can eliminate the function calls and do this:
```suggestion
                  <chef-td>{{ version.key }}</chef-td>
                  <chef-td>{{ version.value.operator }}</chef-td>
                  <chef-td>{{ version.value.versionNumber }}</chef-td>
```